docs/sources/helm-charts/mimir-distributed/get-started-helm-charts/_index.md

```diff
@@ -63,26 +79,136 @@ Using a custom namespace solves problems later on because you do not have to ove
+   mimir-rollout-operator-76c67c7d56-v6xtl     1/1     Running     0          2m44s
+   mimir-nginx-858455979c-hjvhx                1/1     Running     0          2m44s
+   mimir-make-minio-buckets-svgvd              0/1     Completed   1          2m44s
+   mimir-ruler-64b9d59b94-tvj7z                1/1     Running     0          2m44s
+   mimir-query-frontend-c444b56f9-jrmwl        1/1     Running     0          2m44s
+   mimir-overrides-exporter-86c4d54645-zktkm   1/1     Running     0          2m44s
+   mimir-querier-5d9c55d6d9-l6fdc              1/1     Running     0          2m44s
+   mimir-distributor-7796db494f-rsvdx          1/1     Running     0          2m44s
+   mimir-query-scheduler-d5dccfff7-5c5rw       1/1     Running     0          2m44s
+   mimir-querier-5d9c55d6d9-xghl6              1/1     Running     0          2m44s
+   mimir-query-scheduler-d5dccfff7-vz4vf       1/1     Running     0          2m44s
+   mimir-alertmanager-0                        1/1     Running     0          2m44s
+   mimir-store-gateway-zone-b-0                1/1     Running     0          2m44s
+   mimir-store-gateway-zone-c-0                1/1     Running     0          2m43s
+   mimir-ingester-zone-b-0                     1/1     Running     0          2m43s
+   mimir-compactor-0                           1/1     Running     0          2m44s
+   mimir-ingester-zone-a-0                     1/1     Running     0          2m43s
+   mimir-store-gateway-zone-a-0                1/1     Running     0          2m44s
+   mimir-ingester-zone-c-0                     1/1     Running     0          2m44s
+   ```
+
+1. Wait until all of the pods have a status of `Running` or `Completed`, which might take a few minutes.
+
+## Generate some metrics for testing
+
+The Grafana Mimir Helm chart can collect metrics or logs, or both, about Grafana Mimir itself. This is called _metamonitoring_.
+In the example that follows, metamonitoring scrapes metrics about Grafana Mimir itself, and then writes those metrics to the same Grafana Mimir instance.
 
-   b. Add the following configuration to the file:
+1. Create a YAML file called `custom.yaml` for your Helm values.
+
+1. To enable metamonitoring in Grafana Mimir, add the following YAML snippet to your Grafana Mimir `custom.yaml` file:
+
+   ```yaml
+   metaMonitoring:
+     serviceMonitor:
+       enabled: true
+     grafanaAgent:
+       enabled: true
+       installOperator: true
+       metrics:
+         additionalRemoteWriteConfigs:
+           - url: "http://mimir-nginx.mimir-test.svc:80/api/v1/push"
+   ```
+
+   > **Note:** In a production environment the `url` above would point to an external system, independent of your Grafana Mimir instance, such as an instance of Grafana Cloud Metrics.
+
+1. Upgrade Grafana Mimir by using the `helm` command:
+
+   ```bash
+   helm -n mimir-test upgrade mimir grafana/mimir-distributed -f custom.yaml
+   ```
```

Comment:
Before doing that they should probably install the [grafana agent operator CRDs](https://github.com/grafana/helm-charts/tree/main/charts/agent-operator/crds) in the cluster.

Reply:
Interestingly neither @osg-grafana or me had to do that. The agent operator installed the CRDs. I tested in microk8s that if I delete one (e.g. GrafanaAgent) the subchart simply installed it.